web/themes/custom/jcc_elevated/templates/paragraphs/paragraph--steps.html.twig

```diff
@@ -0,0 +1,47 @@
+{{ attach_library("jcc_storybook/Steps") }}
+{{ attach_library("jcc_storybook/Container") }}
+{{ attach_library("jcc_storybook/Stack") }}
+
+{% set classes = [
+  'paragraph',
+  'paragraph__steps',
+  'box'
+] %}
+
+<div class="{{ classes|join(" ") }}">
+  <div class="container stack">
+
+    {% if content.field_lead or content.field_heading %}
```

Comment:
I will also call this out here.  If these things are meant to be part of the Steps component, they should be added to the component in storybook, not here.

Reply:
Steps template has been completely redone.